onnxruntime/core/providers/openvino/openvino_execution_provider.cc

```diff
@@ -19,23 +19,62 @@ OpenVINOExecutionProvider::OpenVINOExecutionProvider(const OpenVINOExecutionProv
   openvino_ep::BackendManager::GetGlobalContext().use_compiled_network = info.use_compiled_network_;
   openvino_ep::BackendManager::GetGlobalContext().blob_dump_path = info.blob_dump_path_;
   openvino_ep::BackendManager::GetGlobalContext().context = info.context_;
+  openvino_ep::BackendManager::GetGlobalContext().enable_opencl_throttling = info.enable_opencl_throttling_;
+
 
   if ((int)info.num_of_threads_ <= 0) {
     openvino_ep::BackendManager::GetGlobalContext().num_of_threads = 8;
   } else {
     openvino_ep::BackendManager::GetGlobalContext().num_of_threads = info.num_of_threads_;
   }
+  //to check if target device is available
+  //using ie_core capability GetAvailableDevices to fetch list of devices plugged in
+  bool device_found = false;
+  bool device_id_found = false;
+  auto available_devices = openvino_ep::BackendManager::GetGlobalContext().ie_core.GetAvailableDevices();
```

Comment:
Is there a way to silence the output from this call. If the inference takes long, the user only sees below for a little while: 
`[E:] [BSL] found 0 ioexpander device`

Reply:
unfortunately, there is not much we can do to disable this as its coming from the OpenVINO API's. I will create a ticket with OpenVINO team to see if we can get this print removed from the next OpenVINO release (2022.2)